doc/learn/syntax-trees-with-typescript.md

```diff
@@ -0,0 +1,523 @@
+---
+group: guide
+title: Working with syntax trees in TypeScript
+description: Guide that shows how to traverse, update, and create syntax trees in TypeScript
+author: Christian Murphy
+authorGithub: ChristianMurphy
+tags:
+  - typescript
+published: 2020-06-09
+modified: 2020-06-09
+---
+
+## Working with syntax trees in TypeScript
+
+This guide will introduce you to:
```

Comment:
If you add a `### Contents`, the table will be automatically generated.

Reply:
good idea, added